hadoop-hdds/tools/src/main/java/org/apache/hadoop/hdds/scm/cli/ContainerBalancerStartSubcommand.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.hadoop.hdds.scm.cli;
+
+import org.apache.hadoop.hdds.cli.HddsVersionProvider;
+import org.apache.hadoop.hdds.scm.client.ScmClient;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+
+import java.io.IOException;
+
+/**
+ * Handler to start container balancer.
+ */
+@Command(
+    name = "start",
+    description = "Start ContainerBalancer",
+    mixinStandardHelpOptions = true,
+    versionProvider = HddsVersionProvider.class)
+public class ContainerBalancerStartSubcommand extends ScmSubcommand {
+
+  @CommandLine.Option(names = {"-t", "--threshold"},
+      description = "Threshold target whether the cluster is balanced")
+  private double threshold;
```

Comment:
We can specify all these options as Optional<Double> etc. The SCMClient could also use Optional as parameter.

Reply:
thanks @lokeshj1703 for pointing out this! but ,according to my experience , it is not a good choice to use Optional as parameter, see [Java static code analysis](https://rules.sonarsource.com/java/RSPEC-3553), so i use another approach to make it optional, and add some checks to validate the input parameters. please take a look and correct me if i misunderstand.

Reply:
Sure, this works too.

Optional would work in our case because we are mostly passing these params if present to a protobuf or a configuration object. It might make code simpler IMO. But I am fine with the above approach too.